docs/sink-configuration.md

```diff
@@ -184,6 +184,7 @@ Besides this, the following options can be set in query string:
   a response from Elasticsearch on startup, i.e. when creating a client. The
   default value is `1`.
 * `bulkWorkers` - number of workers for bulk processing. Default value is `5`.
+* `cluster` - cluster name for different Kubernetes clusters. Default value is `default`.
```

Comment:
I think we should call it `cluster_name` and not just cluster.. it'll be easier to understand imo